mer_verify_kernel_config

```diff
@@ -41,22 +70,55 @@ while (<DATA>) {
   my @allowed = parse_line(",", 1, $allowed);
 
   my $warning;
+  my $version;
   # Strip leading/trailing space for each value and check for warnings
   foreach (@allowed) {
     s/^\s+|\s+$//g;
     $warning = 1 if $_ eq "!" ;
+    $version = $_ if m/[\d\.<>=]/;
   }
 
   # Each CONFIG_* has an array of allowed values, a comment and a flag
   # to say it's only a warning (in which case we print the comment)
   $config{$conf} = {allowed => \@allowed,
-		    comment => $comment,
-		    warning => $warning };
+                    comment => $comment,
+                    warning => $warning,
+                    version => $version};
 }
 
+
 print "\nScanning\n" if $debug;
 while (<>) {
+  my $kconfig_line = 0;
+  my $kernel_version="";
```

Comment:
I think these two will be reset on every loop. Suggest moving $kconfig_line up two lines. I don't think $kernel_version really needs to be in this scope - it can be declared as it's assigned on line 99. The eq check on line 95 isn't really necessary.